bs-react-native-next/src/components/ImageBackground.re

```diff
@@ -0,0 +1,57 @@
+type t;
+
+[@bs.obj]
+external style:
+  (
+    ~backfaceVisibility: [@bs.string] [ | `visible | `hidden ]=?,
+    ~backgroundColor: Style.color=?,
+    ~borderColor: Style.color=?,
+    ~borderRadius: float=?,                 
+    ~borderBottomLeftRadius: float=?,
+    ~borderBottomRightRadius: float=?,
+    ~borderTopLeftRadius: float=?,
+    ~borderTopRightRadius: float=?,
+    ~borderWidth: float=?,
+    ~opacity: float=?,
+    ~overflow: [@bs.string] [ | `visible | `hidden ]=?,
+    ~resizeMode: [@bs.string] [ | `cover | `contain | `stretch | `repeat | `center ]=?,
+    ~overlayColor: Style.color=?,
+    ~tintColor: Style.color=?,
+    unit
+  ) =>
+  t =
+  "";
+
+[@react.component] [@bs.module "react-native"]
+external make:
+  (
+    ~onLayout: Event.NativeLayoutEvent.t => unit=?,
+    ~onLoad: unit => unit=?,
+    ~onLoadEnd: unit => unit=?,
+    ~onLoadStart: unit => unit=?,
+    ~resizeMode: [@bs.string] [
+                   | `center
+                   | `contain
+                   | `cover
+                   | `repeat
+                   | `stretch
+                 ]
+                   =?,
+    ~source: Image.Source.t,
+    ~style: Style.t=?,
+    ~imageStyle: t=?,
```

Comment:
Could you add imageRef too ? https://facebook.github.io/react-native/docs/imagebackground#imageref

Reply:
Sure, will take care of that later tonight